pull_request_template.md

```diff
@@ -6,15 +6,22 @@ Describe the big picture of your changes here to communicate to the maintainers
 
 _Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
 
-- [ ] PR is well described and the description can be used as commit message on squash
+**Contributor**:
+- [ ] PR is well described and the description can be used as a commit message on squash
 - [ ] Related issues linked to PR if existing and labels set
 - [ ] Target branch is not master (in most cases develop should bet the target of choice) 
 - [ ] Code does not conflict with target branch
-- [ ] New code is covered with unit tests
+- [ ] New code is covered with unit tests (coverage >= 80 %)
```

Comment:
I'm not a fan of hard limits for coverage. In some cases it does not make sense to force the 80%,